pkg/asset/installconfig/aws/validation.go

```diff
@@ -328,7 +330,7 @@ func validateEndpointAccessibility(endpointURL string) error {
 	if port == "" {
 		port = "https"
 	}
-	conn, err := net.Dial("tcp", net.JoinHostPort(URL.Hostname(), port))
+	conn, err := proxy.Dial(ctx, "tcp", net.JoinHostPort(URL.Hostname(), port))
```

Comment:
I don't see much reason to use a `ctx` variable.
```suggestion
	conn, err := proxy.Dial(context.Background(), "tcp", net.JoinHostPort(URL.Hostname(), port))
```